aws-lc-rs/src/aead.rs

```diff
@@ -392,11 +391,11 @@ impl<N: NonceSequence> Debug for SealingKey<N> {
 impl<N: NonceSequence> SealingKey<N> {
     /// Deprecated. Renamed to `seal_in_place_append_tag`.
     ///
-    /// # FIPS
-    /// This method must not be used.
-    ///
-    /// See [`RandomizedNonceKey::seal_in_place_append_tag`].
-    ///
+    // # FIPS
+    // This method must not be used.
+    //
+    // See [`RandomizedNonceKey::seal_in_place_append_tag`].
```

Comment:
Here and in other places, should our documentation continue to reference a "preferred"  `RandomizedNonceKey` method? Possibly changing the wording to "Prefer..." instead of "See..." or "Use..."?